### ISSUE TITLE
Entry text flickering while typing

### ISSUE THREAD

**Issue by jojosboomstick** [fyne-io/fyne#4635]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

This issue has already been discussed, but it looks a little different. Perhaps this is important.
When tapping on an Entry widget, text is slightly flickering.

### How to reproduce

Just start typing on widget.NewEntry

### Screenshots

![final](https://github.com/fyne-io/fyne/assets/66790655/4b1ea33c-2d98-420a-92bd-26b0c2fdc116)

### Example code

    package main

    import (
    	"fyne.io/fyne/v2"
    	"fyne.io/fyne/v2/app"
    	"fyne.io/fyne/v2/container"
    	"fyne.io/fyne/v2/theme"
    	"fyne.io/fyne/v2/widget"
    )

    func main() {
    	a := app.New()
    	w := a.NewWindow("")
    	w.Resize(fyne.NewSize(640, 480))
    	w.CenterOnScreen()
    	inp := widget.NewEntry()
    	inp.MultiLine = false
    	w.SetContent(container.NewVBox(inp))
    	w.ShowAndRun()
    }

### Fyne version

2.4.3

### Go compiler version

1.21.1

### Operating system and version

Windows 10 Pro ver 10.0.19045.3086

### Additional Information

nVIDIA GeForce GT 730
driver version	23.21.13.9135 - nVIDIA ForceWare 391.35

**Comment by andydotxyz** [comment]
Given that v2.4.4 is out now can you please update to that and quickly see if it makes a difference?

**Comment by jojosboomstick** [comment]
Unfortunately the problem remains.
I also checked on two other computers with Windows 10:
On a PC with Nvidia GTX 560 and laptop Lenovo v14ada with AMD Ryzen 3 3250U.
I also checked with other versions of fyne and here's what's interesting
2.1.0 all ok
2.3.0 all ok
2.3.3 all ok
2.3.4 - slightly different glitch effect
2.3.5 - slightly different glitch effect
2.4.0 - 2.4.4 - the same effect as in gif
The faster you press the buttons, the more often the flickering appears.
